tests/onnx2circom/test_onnx_to_circom.py

```diff
@@ -14,8 +14,8 @@
 
 
 # NOTE: Change the path to your own path
-CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/path/to/circom-2-arithc-project-root')
-MP_SPDZ_PROJECT_ROOT = Path('/path/to/mp-spdz-project-root')
+CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/Users/jernkun/circom-2-arithc')
```

Comment:
should we changed these back to 

```python
CIRCOM_2_ARITHC_PROJECT_ROOT = Path('/path/to/circom-2-arithc-project-root')
MP_SPDZ_PROJECT_ROOT = Path('/path/to/mp-spdz-project-root')
```
or we can add a `tests/onnx2circom/.env` to store the paths, and load them in the tests with [python-dotenv](https://pypi.org/project/python-dotenv/)

Reply:
Thanks!! Resolved :))